py-rse/ci.Rmd

```diff
@@ -287,47 +251,49 @@ after_success:
 - codecov
 ```
 
+We use the Travis setting `after_success` since the coverage tool is a separate
+service and the Travis build should happen first.
 We use [`codecov`][codecov] rather than the `coverage` tool described in Section \@ref(py-rse-correct-coverage)
-because it is a service like Travis CI.
+since it is a service like Travis CI while the `coverage` tool is a program.
 
-FIXME: fill in this explanation
+Go to Exercise \@ref(py-rse-ci-ex-coverage) and complete it to add it to the
+Zipf's project.
 
-## How can I use CI for other purposes? {#py-rse-ci-deploying}
+## Display the Travis status on your README {#py-rse-ci-display-github}
 
-Continuous integration was invented for testing,
-but it can be used to automate almost anything.
-For example,
-if we want to create a website for an R package,
-we can run:
+Travis CI's dashboard is very useful,
+but is also convenient to display the status of the build on our project's GitHub home page
+(which most people look at more often).
+To add a status badge,
+click the "build icon" shown in the top right corner of Figure \@ref(fig:image-github-icon)
+to bring up a dialog box.
+Select "Markdown" from the "Format" menu list,
+then copy the Markdown text displayed in the "Result" box
+and paste it into the project's `README.md` file.
+(It's best to paste the text right below the page's title so that it will be instantly visible).
+Complete Exercise \@ref(py-rse-ci-ex-badge) to add the badge to your README file.
+When you've added it,
+your project will now show the badge for its Travis CI status as shown in Figure \@ref(fig:py-rse-ci-github-icon).
 
-```r
-usethis::use_pkgdown()
-usethis::use_pkgdown_travis()
+```{r py-rse-ci-github-icon, echo=FALSE, fig.cap="The Travis CI Build Badge"}
+knitr::include_graphics("figures/rse-ci/github-icon.png")
 ```
 
-to add lines like this to our `.travis.yml` file:
-
-```yaml
-language: R
-sudo: false
-cache: packages
-
-before_cache: Rscript -e 'remotes::install_cran("pkgdown")'
-deploy:
-  provider: script
-  script: Rscript -e 'pkgdown::deploy_site_github()'
-  skip_cleanup: true
-```
+> **How Do Badges Work?**
+>
+> Badges are much simpler than they first appear.
```

Comment:
Since we don't know the reader personally, we don't know if this claim is true, and if it isn't, it will feel like a "just".

Reply:
I added a short description of what a badge is to the previous paragraph and removed this.